RELEASENOTES.md

```diff
@@ -4,11 +4,15 @@ Releases, starting with 9/2/2021, are listed with the most recent release at the
 
 ## NuGet Version 0.100.5
 
-__API Changes__:
+__Breaking Changes__:
 
+Inplace operators no longer create an alias, but instead return 'this'. This change will impact any code that explicitly calls `Dispose` on a tensor after the operation.
```

Comment:
Is the expected impact a noop, or exception?

Reply:
The expected impact is that if you use dispose scopes, then you don't worry about anything. If you explicitly clean up with calls to `Dispose()` you have to worry about this:

Before: 
```C#
Tensor x = ...'
Tensor y = x.mul_(10);
x.Dispose();
-- y can now be used, the Dispose() call just decremented the native reference counter --
```

After:
```C#
Tensor x = ...'
Tensor y = x.mul_(10);
x.Dispose();
-- y has been disposed, because it's the same managed code instance as X --
```

Reply:
You can address this by introducing explicit calls to `.alias()`, as I had to do in a couple of places in the code.

Reply:
It might be worth adding this to [Memory.md](https://github.com/dotnet/TorchSharp/blob/main/docfx/articles/memory.md) and referencing it in the releasenotes?

Reply:
Something a little longer will be added to the relevant wiki page once released.